index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Aggregation App</title>
+	<meta name="theme-color" content="#151828">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cabin" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+	<header>
+        <a href="/">
+            <img src="https://raw.githubusercontent.com/NguyenD-Nam/Aggregation-app/main/public/icons.png" height="50px"><h1>Aggregation app</h1>
+        </a>
+    </header>
+    <div class="tool__bar">
+        <div>
+            <input id="input" type="text" placeholder="octocat, dwarvesf, etc." autocomplete="off">
+            <button onclick="getInputVal()"><i class="fas fa-search"></i></button>
+        </div>
+        <div>
+            <button class="sort__btn" onclick="setSortCriteria('stars')">Stars</button>
+            <button class="sort__btn" onclick="setSortCriteria('popular')">Popular</button>
+            <button class="sort__btn" onclick="setSortCriteria('original')">Original</button>
```

Comment:
Should not use inline `onclick` in HTML document. Please move the call into the script file.